package.json

```diff
@@ -16,12 +16,15 @@
   "scripts": {
     "lint": "eslint .",
     "lint:report": "eslint -f checkstyle -o build2/reports/lint/eslint-checkstyle-result.xml .",
-    "test": "yarn test:browser && yarn test:server",
+    "test": "yarn test:karma && yarn test:browser && yarn test:server",
+    "test:karma": "karma start --single-run",
     "test:browser": "jest --config ./jest.browser.js",
     "test:server": "jest --config ./jest.server.js",
     "test:report": "yarn test --ci --silent || true",
     "build": "node ./writeConfig.js && webpack --config webpack.config.js",
-    "prepublish": "yarn build"
+    "prepare": "yarn build",
+    "prestart": "npm install -C test/app",
```

Comment:
Interesting.. what does `-C` do? I couldn't find any documentation on this.

Reply:
This is an alias for the --prefix option, so it only runs in that directory.

Reply:
I can change this to `yarn install --cwd test/app` but for some reason it takes MUCH longer. It seems to be examining the parent folder.